kimchi/src/circuits/expr.rs

```diff
@@ -2220,6 +2305,7 @@ where
                 cache.insert(*id, e.as_ref().clone());
                 id.latex_name()
             }
+            EnabledIf(feature, _) => format!("{:?}", feature),
```

Comment:
you should recurse here no?

Reply:
The idea here is that we declare the terms opaquely here, then state the full constraints elsewhere, so that we're not inlining potentially disabled constraints.